vignettes/User-input-example.Rmd

```diff
@@ -132,21 +136,17 @@ signatures greatly overlap with others.
 `plot_signature_overlap` creates a heatmap of odds ratios. The odds ratio
 represents the odds that two signatures share a specific number of genes.
 
-```{r signature overlap}
-plot_signature_overlap(selected_signatures = active_sigs)
+```{r}
+signature_similarity <- plot_signature_overlap(selected_signatures = active_sigs)
```

Comment:
Yes, it will still display the plot and return a reordered matrice at the same time.